src/app/PCM/templates/tpl-sidebar-one/index.js

```diff
@@ -5,14 +5,14 @@ import InternmatchSidebar from 'app/PCM/templates/tpl-sidebar-one/internmatch'
 import { useIsProductInternmatch } from 'utils/helpers/check-product-name'
 
 const TemplateSidebarOne = ({ mappedPcm, maxItemCount, isSidebarCollapsed }) => {
-  const maxItems = maxItemCount || Math.floor(window.innerHeight - 60) / 100
+  const maxItems = maxItemCount || Math.floor(window.innerHeight - 60) / 95
```

Comment:
let's start declaring a variable that changes its value with let rather than const. Nothing wrong with declaring it with const but we will move towards more declarative approach going forward.